client/components/SignIn.jsx

```diff
@@ -1,20 +1,23 @@
 import React from 'react'
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button'
 
 export default function SignIn() {
   return (
     <div>
-      <h1 className="signin">SignIn</h1>
-      <form className="form">
-        <span>Username </span>
-        <input type="text" name="username" />
-        <br />
-        <br />
-        <span>Password </span>
-        <input type="text" name="password" />
-        <br />
-        <br />
-        <input type="submit" value="Submit" />
-      </form>
+      <h1>SIGN IN COMPONENT</h1>
```

Comment:
Remove please, we do not need this in our dev branch as the user will not see this.